pynamodb/attributes.py

```diff
@@ -372,6 +382,43 @@ def _deserialize(self, attribute_values: Dict[str, Dict[str, Any]]) -> None:
                 value = attr.deserialize(attr.get_value(attribute_value))
                 setattr(self, name, value)
 
+    def to_json(self) -> str:
```

Comment:
Does it produce a JSON in the DynamoDB format of `{S: "foo"}`? From the tests it doesn't seem this way. Maybe it'll be beneficial, and also help with code reuse?

Reply:
no, this renders primitive values -- i'm not sure how useful serializing with the dynamo types would be -- and of course if the user wants that they can already do that with:

```
json.dumps(AttributeContainer._serialize(model))
```

My thinking here is that we already support serializing to dynamo types.
If you want a python dictionary with the python values themselves you can get that via `item.attribute_values`.
If you want a python dictionary with the serialized values (so you get JSON primitives) you can now do that via
`json.loads(item.to_json())`